pkg/dataplane/http/context.go

```diff
@@ -1719,11 +1719,14 @@ func (c *context) PutOOSObjectSync(putOOSObjectInput *v3io.PutOOSObjectInput) er
 		buffer.Write(ioVec)
 	}
 
-	headers := putOOSObjectHeaders
-	headers["slice"] = strconv.Itoa(putOOSObjectInput.SliceID)
-	headers["io-vec-num"] = strconv.Itoa(len(putOOSObjectInput.Data) + 1)
-	headers["io-vec-sizes"] = iovecSizes.String()
-
+	// headers for OOS put object
+	headers := map[string]string{
+		"Content-Type":    putOOSObjectHeaders["Content-Type"],
+		"X-v3io-function": putOOSObjectHeaders["X-v3io-function"],
+		"slice":           "0",
```

Comment:
please check if you can remove the slice header from here and nginx - i believe the new get_slice_id() logic in v3io should make the code ignore the slice header anyway..

Reply:
will do